config/locales/en.yml

```diff
@@ -73,6 +76,18 @@ en:
     log_out: "Exit"
     logged_out: "Your session has been cleared."
     new: "Submit contact information"
+    new_auth_token: "Log in with email"
+    new_auth_token_header: "Existing publisher login"
+    new_auth_token_body: "If you've already started or completed verification, we can send a login link to the email address associated with your publisher registration."
+    new_auth_token_body_2: "If you've forgotten the email:"
+    new_auth_token_body_3: "1) For verifications completed, leave email blank and we'll use the most recent email."
+    new_auth_token_body_4: "2) For verifications in progress, please "
```

Comment:
While the programmer in me loves how the two entries above line up (For verifications X, Y) this sounds a little unnatural- maybe it could be:
`1) If you've already completed the verification process, you can leave email blank and we'll use the most recent one on file`
`2) If you were still working through verification, please include email or `
`start a new verification`